components/create-collective/sections/ConnectGithub.js

```diff
@@ -0,0 +1,406 @@
+                    width={1 / 5}
+                    minWidth={[200, 335]}
+                  />
+                </Flex>
+              </Fragment>
+            )}
+          </Fragment>
+        )}
+        {!token && (
+          <Fragment>
+            <Flex flexDirection="column" my={[2, 4]}>
+              <Box textAlign="left" minHeight={['32px']} marginLeft={['none', '224px']}>
+                <StyledButton
+                  asLink
+                  fontSize="Paragraph"
+                  color="black.600"
+                  onClick={() => {
+                    this.changeRoute({ category: undefined });
+                    this.handleChange('category', null);
+                  }}
+                >
+                  ←&nbsp;{intl.formatMessage(this.messages.back)}
+                </StyledButton>
+              </Box>
+              <Box mb={[2, 3]}>
+                <H1
+                  fontSize={['H5', 'H3']}
+                  lineHeight={['H5', 'H3']}
+                  fontWeight="bold"
+                  textAlign="center"
+                  color="black.900"
+                >
+                  {intl.formatMessage(this.messages.header)}
+                </H1>
+              </Box>
+              <Box textAlign="center" minHeight={['24px']}>
+                <P fontSize="Paragraph" color="black.600" mb={2}>
+                  {intl.formatMessage(this.messages.openSourceSubtitle)}
+                </P>
+              </Box>
+            </Flex>
+            {error && (
+              <Flex alignItems="center" justifyContent="center">
+                <MessageBox type="error" withIcon mb={[1, 3]}>
+                  {error}
+                </MessageBox>
+              </Flex>
+            )}
+            <Flex alignItems="center" justifyContent="center">
+              <Box mb={[1, 5]} minWidth={['300px', '576px']} maxWidth={[288, 608, 576, null]} px={[1, 4]}>
+                <P mb={3}>
+                  <FormattedMessage
+                    id="createcollective.opensource.p1"
+                    defaultMessage="You're creating software. You don't want to worry about creating a legal entity or seperate bank account, paying taxes, or providing invoices to sponsors. Let us take care of all that, so you can stay focused on your project."
+                  />
+                </P>
+                <P mb={3}>
+                  <FormattedMessage
+                    id="createcollective.opensource.p2"
+                    defaultMessage="We have created the {osclink}, a non-profit umbrella organization, to serve the open source community. To join, you need at least 100 stars on Github or meet our {criterialink}."
+                    values={{
+                      osclink: (
+                        <a href="https://opencollective.com/opensource" target="_blank" rel="noopener noreferrer">
+                          Open Source Collective 501c6
+                        </a>
```

Comment:
I'd suggest using `ExternalLink`, it's made to avoid the repetition with `target="blank"`. Great that you also thought about using the `rel="noopener noreferrer"` tho!

```suggestion
                        <ExternalLink href="https://opencollective.com/opensource" openInNewTab>
                          Open Source Collective 501c6
                        </ExternalLink>
```